src/molecules/formfields/CreditCardField/credit_card_field.module.scss

```diff
@@ -1,45 +0,0 @@
-$form_border: 1px solid color('neutral-5');
```

Comment:
I don't think we would need to reference two sets of styles. If we turn the shared classes into placeholders, then we can import that shared file into the individual scss files and use the placeholders as necessary